packages/docz-tools/src/gatsby-theme-docz/components/Actions/Actions.tsx

```diff
@@ -24,15 +28,26 @@ export function Actions() {
               // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
               // @ts-ignore
               label={<Github />}
-              url={doc.link}
+              url={repository}
               external
-              type="secondary"
+              type="tertiary"
```

Comment:
I think we should leave this as `secondary` since we're already exploring moving to page, this change would only live for a short time
![image](https://user-images.githubusercontent.com/39704901/111842623-59b68200-88c5-11eb-950c-571cf7e3a15c.png)